gateway/ln-gateway/src/events.rs

```diff
@@ -22,4 +31,263 @@ pub const ALL_GATEWAY_EVENTS: [EventKind; 11] = [
     DepositConfirmed::KIND,
 ];
 
-// TODO: Add Gateway specific events
+pub type LogEntry = (
```

Comment:
That's... a weird way to write `struct LogEntry`... :D

Reply:
Fixed. Moved and renamed to `PersistedLogEntry` to not be confused with `EventLogEntry`